turbo.json

```diff
@@ -1,5 +1,4 @@
 {
-  "$schema": "https://turbo.build/schema.json",
```

Comment:
Hot take (not blocking) We should leave this here even though we've set the vscode override to be an example for anyone looking for the schema url.

Reply:
ok, I see what you mean.  yeah, it's probably better that way.  ultimately, it looks like we're gonna be trying to just redirect `https://turbo.build/schema.json` directly to this file (hosted here on GitHub) so that means it's gonna work (soon) both ways, so that's fine.  and besides, with the `vscode` setting in place, if we simply remove this schema, we'll see the local one so it's almost like the best of both worlds.

added back in 8ece8cdacee8200812798f2de3aaa765fe111951